src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -128,7 +128,7 @@ export const ShowcaseLayout = ({ CAPI, config, NAV }: Props) => (
         </Section>
 
         <Section islandId="most-viewed">
-            <MostViewed
+            <MostViewedGrid
```

Comment:
This was removed in the `StandardLayout` so we should also drop it here. Or, we could perhaps separate the removal of the component into a different PR?

Reply:
I've dropped MostViewedGrid here for consistency with StandardLayout.